RInAppMessaging/Classes/InAppMessagingModule.swift

```diff
@@ -164,3 +166,12 @@ extension InAppMessagingModule {
         return randomizer.dice <= config.rolloutPercentage
     }
 }
+
+// MARK: - UserChangeObserver
+extension InAppMessagingModule {
+
+    func didUserChangeOrLogout() {
```

Comment:
didUser -> userDid

Is it a checking function or is dealing with a known change of state?

Reply:
`userDid` is better. It's dealing with a known change